tests/support/globalSetup.ts

```diff
@@ -37,14 +36,11 @@ export default async function globalSetup(_config: FullConfig): Promise<void> {
     }
     await removeWorkspace();
 
-    // Workaround for fixing MaxListenersExceededWarning.
-    // Since every prefixOutput stream adds stream listeners that don't autoClose.
-    // TODO: We still seem to have some descriptors left open when running vitest, which we should handle.
     const peerManager = await createPeerManager({
       dataDir: Path.resolve(Path.join(tmpDir, "peers")),
       outputLog: FsSync.createWriteStream(
         Path.join(tmpDir, "peerManager.log"),
-      ).setMaxListeners(20),
+      ).setMaxListeners(16),
```

Comment:
Do we still need this at all?

Reply:
yes, sadly we still need it, not for the reasons in the comment, so I removed it but we still get the warning that we have more than 15 max listeners added.
This is a default by node

> By default EventEmitters will print a warning if more than 10 listeners are added for a particular event. This is a useful default that helps finding memory leaks. The emitter.setMaxListeners() method allows the limit to be modified for this specific EventEmitter instance. The value can be set to Infinity (or 0) to indicate an unlimited number of listeners.
source: https://nodejs.org/api/events.html#emittersetmaxlistenersn